gravitee-apim-console-webui/src/services-ngx/identity-provider.service.ts

```diff
@@ -32,14 +33,21 @@ export class IdentityProviderService {
     return this.http.get<IdentityProviderListItem[]>(`${this.constants.org.baseURL}/configuration/identities`);
   }
 
+  listActivated(): Observable<IdentityProviderActivation[]> {
```

Comment:
Cloud you add this two method into a dedicated service like `EnvironmentIdentityProviderService` 
this one are used for Organization

Reply:
Done